exercises/Graphing-sexual-dimorphism-data-manipulation-R.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: exercise
+topics: Graphing
+title: Sexual Dimorphism Data Manipulation
+language: R
+---
+
+This is a follow up to [Sexual Dimorophism Exploration]({{ site.baseurl }}/exercises/Graphing-sexual-dimorphism-exploration-R).
+
+Having done some basic visualization of the
+[Lislevand et al. 2007](https://doi.org/10.1890/06-2054)
+[dataset of bird size measures](http://www.esapubs.org/archive/ecol/E088/096/avian_ssd_jan07.txt)
+you realize that you'll need to do some data manipulation to really get at the
+questions you want to answer.
+
+1. In [Sexual Dimorophism Exploration]({{ site.baseurl }}/exercises/Graphing-sexual-dimorphism-exploration-R)
+you created a plot of the histograms of female and male masses by family. This
+resulted in a lot of plots, but many of them had low sample sizes.
+
+The following code creates a list of families along with counts of how many
```

Comment:
Mostly I was hung up by the link to the next line and "familie non-null masses" ?